apps/backend/src/mcp/tools/device_info.py

```
@@ -203,6 +200,775 @@ def _parse_smart_output(smart_output: str, drive_path: str) -> Dict[str, Any]:
+        drive_health["error"] = str(e)
+        logger.warning(f"Drive health analysis failed: {e}")
+
+    return drive_health
+
+
+async def _collect_processes(
+    ssh_client, connection_info: SSHConnectionInfo
+) -> list:
+    """Collect top processes information."""
+    processes = []
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)